doc/developer/interface/webpack/index.md

```diff
@@ -2,11 +2,55 @@
 layout: default
 ---
 
-
 Webpack
```

Comment:
I think I'd like to see a lot more documentation in here about this feature.  A lot about how it works and what it's achieving.  I see some stuff below about the tags that you need to add, but maybe about what those tags are replacing as well.  Maybe ways to work around it?  Special cases?

Overall we need much more doc on UI development for IRIDA in general, and given that this is an important part of the UI build scheme we should bump this up here too.